src/platforms/native/index.mdx

```diff
@@ -93,7 +88,7 @@ event data may be lost.
 Alternatively, the DSN can be passed as `SENTRY_DSN` environment variable during
 runtime. This can be especially useful for server applications.
 
-### Verifying Your Setup
+### Verify
 
 Now that SDK setup is complete, verify that all configuration is correct. Start
 by capturing a manual event:
```

Comment:
Two things here:
1. I'd move "Capturing Events" and everything below to a separate pages. Ironically in native, this is an advanced use case and usually users just capture crashes (auto) initially. 
2. I'd also add a "What's Next" section at the bottom of this page similar to JavaScript.

Reply:
whats next is out of scope for this (mostly moving content) but i moved the other stuff